### ISSUE TITLE
Refueling Pylon Error

### ISSUE THREAD

**Issue by Vando99** [jpw1991/chebs-necromancy#195]
# Describe the bug
When adding materials for it to use after it does it's initial input of the item it throws an error.

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
ChebsNecromancy.Structures.RefuelerPylon.ManageFireplace (Fireplace fireplace) (at <cc1c2b88898449f4bb3d7cfb69aa94e5>:0)
System.Collections.Generic.List`1[T].ForEach (System.Action`1[T] action) (at <695d1cc93cca45069c528c15c9fdd749>:0)
ChebsNecromancy.Structures.RefuelerPylon+<LookForPieces>d__15.MoveNext () (at <cc1c2b88898449f4bb3d7cfb69aa94e5>:0)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <80fe1a4e36fe44618284312cb721a597>:0)

**Comment by jpw1991** [comment]
Hello,

What kind of items are you putting in there and what type of fireplaces do you have?

**Comment by Vando99** [comment]
Figured it out. Turns out the BoneAppetit mod was having a hearth issue that was causing the refueling pylons to act that way. The pylons I was having problems with, where I noticed the error, was the ones trying to add coal and iron to my smelters. They didn't have wood in them so I didn't think much about it being a problem with a fireplace but makes since now I guess. But after they did an update tonight it seems to be working fine now. Thanks for the reply! Your mod is so handy and makes the upkeep a lot more easier. Thank you!
